src/modules/group/group.routes.ts

```diff
@@ -31,6 +30,7 @@ export const createGroupRoute = createRoute({
         status: true,
         createdAt: true,
         updatedAt: true,
+        creatorId: true,
```

Comment:
We do not need the creatorId from the front end.

We will consider the creator as the logged-in user creating the group.

Reply:
Oh, my bad, this is in omit.